app/assets/javascripts/discourse/app/lib/sidebar/admin-nav-map.js

```diff
@@ -159,8 +159,9 @@ export const ADMIN_NAV_MAP = [
         name: "admin_components",
         route: "adminCustomizeThemes",
         routeModels: ["components"],
-        label: "admin.appearance.sidebar_link.components",
+        label: "admin.appearance.sidebar_link.components.title",
         icon: "puzzle-piece",
+        keywords: ["admin.appearance.sidebar_link.components.keywords"],
```

Comment:
Not sure about this, or if we instead do something like `hasKeywords: true` and always assume that the keywords are in the same spot in the translation file based on `name`?

Reply:
Also not sure if I should wrap in array or rely on yml lists?

Reply:
Yeah okay, will leave it here for now. Do you think though we should make them comma separated in the translation then split them when they load? Or maybe they should just be a single space-separated string, that might be enough and we can do away with the array altogether. I guess the only downside is that some links are like:

```
keywords: "theme appearance style some_site_setting_name another_syte_setting_name"
```

Maybe that's fine?

Reply:
I like the array of strings. Especially since later, we may have keywords like "API key for gemini 1.5". If we split this string into individual words, then the search phrase "key for" will not work. Unless we split the input into words and check each word individually (which actually may be not a bad idea). But for now, I think that array of keywords is good enough

Reply:
Okay I fixed it up, ended up with going with a `|` separator and splitting the keywords, I can't use YML lists in translations